code/modules/projectiles/gun_helpers.dm

```diff
@@ -100,12 +24,17 @@
 
 	active_attachable.unload(living_user)
 
+/obj/item/weapon/gun/attackby(obj/item/I, mob/user, params)
+	if(user.get_inactive_held_item() != src || istype(I, /obj/item/attachable) || isgun(I))
+		return ..()
+	reload(I, user)
```

Comment:
out of curiosity why does this not . = ..()

Reply:
i... dont have a reason

Reply:
standing request